app/assets/stylesheets/application.scss

```diff
@@ -26,4 +26,8 @@
 @import 'commontator';
 @import 'announcements';
 @import 'select2.css';
-@import 'select2_overrides';
\ No newline at end of file
+@import 'select2_overrides';
+
+body {
+    min-width: 320px;
+}
```

Comment:
It's not a good idea to set min-width to body. Instead we can try to make other stuffs responsive.
`body` is the top layout, if your device has less than 320px width, the layout will break and show scrollbars everywhere.

Reply:
sir in the body further responsiveness is not possible according to images and text 
and also google and other big org. follows this type of styling you can check youtube by doing so it is better to do this as this is a standard practise

Reply:
If it's not possible, we can skip that. We don't need to squeeze page below 280 px(galaxy fold), we can keep other stuffs as responsive keeping 280px as our target.

Share some official docs (maybe of mdn if available), which can confirm this as a standard practice.

Reply:
so then i apply apply minwidth after 280px till that i will make reponsive

Reply:
is that okay @tanmoysrt